### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- *Upgrade to version 0.18.0*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/searx_ynh%20PR71%20(ericgaspar)/)

### CODE REVIEW

**Comment by HyperCriSiS** [comment]
v18 was released :)

**Comment by AkselRocks** [comment]
@ericgaspar what else does it need for merging into master?

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM

**Comment by ericgaspar** [comment]
The upgrade script is failing with : invalide argument: --
any idears?
ping @Josue-T 

```
Info:  [##++++..............] > Upgrading dependencies...
Info:  [######++............] > Upgrading source files...
Info:  [########+...........] > Upgrading NGINX web server configuration...
Info:  [#########+..........] > Making sure dedicated system user exists...
Info:  [##########++++......] > Upgrading Searx...
Warning: yunohost.hook <lambda> - [963.1] Invalid argument: --
Info:  '/opt/yunohost/searx/lib/python3/site-packages/setuptools' wasn't deleted because it doesn't exist.
Warning: yunohost.hook <lambda> - [963.1] [Error] Upgrade failed.
Info: yunohost.app app_remove - [3547.1] Removing searx...
```

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
CI ok but the page is not diplayed correctly: console is screaming with lots of 404 and errors
`La ressource à l’adresse « https://domain.ltd/searx/searx/static/themes/oscar/css/logicodev.min.css » a été bloquée en raison d’un type MIME (« text/html ») incorrect (X-Content-Type-Options: nosniff).`

😨
!helpme

**Comment by kay0u** [comment]
What, there is no bot for `!helpme` command?!

File: conf/nginx.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ location __PATH__/ {
 	uwsgi_param SCRIPT_NAME '__PATH_NO_ROOT__';
 	include uwsgi_params;
 	uwsgi_modifier1 30;
-	uwsgi_pass unix:///run/uwsgi/app/searx/socket;
+	uwsgi_pass unix:///var/run/__NAME__/app.socket;
```
Comment (kay0u) [inline conf/nginx.conf:13]:
```suggestion

	location __PATH__/searx/static/ {
		alias __FINALPATH__/searx/static/;
	}
```

**Review by kay0u — APPROVED** [review]
Tested on my side

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**